argus/action_manager/windows.py

```diff
@@ -318,13 +318,15 @@ def mkfile(self, path):
             Remote path where the new file should be created.
         """
         if self.is_file(path):
-            LOG.warning("File already exists."
-                        " LastWriteTime and LastAccessTime will be updated.")
+            LOG.warning("File already exists '{}'. LastWriteTime and"
```

Comment:
I think it would look better to log it like this `File '{}' already exists. LastWrite....`